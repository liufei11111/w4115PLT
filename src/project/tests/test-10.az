Int main(Int argc, String argv){
Int a;
a = 3;

Int b;
b = a + 5;
b = 5 + a;

Int c;
a = b - 1;
a = 1 - b;
c = a - b;

c = b;
c = b * 3;
c = a * b;

b = a / 2;
c = a / b;
c = 1 / a;
}