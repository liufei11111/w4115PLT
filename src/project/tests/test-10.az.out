
public static int main(int argc, String argv)
{
try{int a=0;
a = 3;
int b=0;
b = a + 5;
b = 5 + a;
int c=0;
a = b - 1;
a = 1 - b;
c = a - b;
c = b;
c = b * 3;
c = a * b;
b = a / 2;
c = a / b;
c = 1 / a;
}catch(Exception e){return -1;}}

